test/libsolidity/syntaxTests/functionCalls/external_functions_with_calldata_args_assgined_to_function_pointers_of_memory_type.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
```

Comment:
Since this works, we should also have at least one semantic test for such function pointers.